docs/contracts/creating-contracts.rst

```diff
@@ -35,13 +35,14 @@ This means that cyclic creation dependencies are impossible.
 
     pragma solidity >=0.4.22 <0.7.0;
 
+
     contract OwnedToken {
         // `TokenCreator` is a contract type that is defined below.
         // It is fine to reference it as long as it is not used
         // to create a new contract.
-        TokenCreator creator;
-        address owner;
-        bytes32 name;
+        TokenCreator private creator;
```

Comment:
Is that a recommendation?

Reply:
Yes, "explicitly mark visibility of state", shows up for quite a few of our examples.

Reply:
Weird - this shouldn't apply for state variables, because you cannot explicitly mark their visibility - the default visibility has no keyword.

Reply:
Ah sorry, it should be `internal` instead of private.

Reply:
But to be honest, I really would not use a keyword here.

Reply:
@chriseth Ahhh, looks like a lot of these come from the solhint security rules, not the Solidity style rules, so maybe I'll disable this rule for our checks and undo these changes.

Reply:
https://github.com/protofire/solhint/blob/master/docs/rules.md#security-rules